sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -255,16 +265,26 @@ def _convert_featureview_schema_to_milvus_readable(
                             )
                             raise e
 
-            # Appending the above converted values to construct a FieldSchema
-            field_list.append(
-                FieldSchema(
+            field = FieldSchema(
+                name=field_name,
+                dtype=data_type,
+                description=description,
+                is_primary=is_primary,
+                dim=dimensions,
+            )
+
+            if data_type == DataType.VARCHAR:
+                field = FieldSchema(
                     name=field_name,
                     dtype=data_type,
+                    max_length=50,
```

Comment:
What's the reasoning for hardcoding the max length? Is this just a temporary measure?

Reply:
yes, exactly. It is required for varchars and strings apparently are not supported yet in Milvus